src/cli/serve.rs

```diff
@@ -16,6 +16,9 @@ use super::{resolve_path, GlobalOptions};
 const DEFAULT_BIND_ADDRESS: Ipv4Addr = Ipv4Addr::new(127, 0, 0, 1);
 const DEFAULT_PORT: u16 = 34872;
 
+const SYNCING_MODEL_ERR: &str = "Cannot sync a model project file. \
+    Projects must have a `DataModel` at their root to be synced.";
+
```

Comment:
Can we say "serve" instead of "sync"? It's good to keep terminology consistent as much as possible